envs/local.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   NODE_ENV: '"default"',
-  HORIZON_SERVER: '"http://localhost:8000/_/api/"',
-  FILE_STORAGE: '"http://localhost:8000/_/storage/api"',
-  NETWORK_PASSPHRASE: '"TokenD Developer Network"',
+  HORIZON_SERVER: '"https://api.reit.tokend.io"',
```

Comment:
This config is incorrect. Check for right here: https://reit.tokend.io/static/env.js